sql/updates/world/4.3.4/2020_11_20_00_world.sql

```diff
@@ -0,0 +1,2 @@
+INSERT IGNORE INTO `creature_trainer` VALUES (35872, 44, (SELECT gossip_menu_id FROM creature_template WHERE entry  = 35872), 0);
+INSERT IGNORE INTO `creature_trainer` VALUES (44465, 135, (SELECT gossip_menu_id FROM creature_template WHERE entry  = 44465), 0);
```

Comment:
Hey there.
Please clean these queries up and use a delete and insert query with no unnecessary select statements. I There's plenty examples for that in the updates folder.